etc/phpmyadmin/config.inc.php

```diff
@@ -110,6 +117,12 @@
     $cfg['Servers'][$i]['compress'] = false;
     $cfg['Servers'][$i]['AllowNoPassword'] = true;
 }
+for ($i = 1; isset($sockets[$i - 1]); $i++) {
+    if (isset($sockets[$i - 1])) {
```

Comment:
The condition does not make sense here, it's already tested on the loop level. Maybe there should be just one loop with `isset($hosts[$i - 1]) && isset($sockets[$i - 1])` condition and this would be used inside of it?

Reply:
You're right.
I adapted the code dealing with $hosts but in $sockets case, it's pointless.